### DISCUSSION TITLE
How to update the app without having to reinstall

### DISCUSSION THREAD

**Discussion by georganix** [TransformerOptimus/SuperAGI#425] [Q&A]
How to update the app without having to reinstall. Can we get an update notification and one click update button?

**Comment by KeulzZz** [comment]
Take what i say with a grain of salt, i'm a noob, but from what i know, dockers are not meant to be updated. If you want to change version, you delete the last one and start up the new one. That simplifies a lot of thing to always start fresh

**Comment by DMTarmey** [comment]
well in docker there is ability to make a pull request from the main file if you make an update on the file system being used so you do not have to pull hole project down. https://www.youtube.com/watch?v=29-BVcbytYc
